### PR TITLE
Restore old blogs from podman_old.

### PR DESCRIPTION
**Pull request by cgiradkar**
Following changes are done in this PR: 
- All the blog posts from _posts folder (except RAs) , from old podman repo, have been moved to blog folder in this repo.
- For docusaurus to work most efficiently, all the plugins which are used should have same version, so made all plugins' version to be 2.4.1

Signed-off-by: Chetan Giradkar <cgiradka@redhat.com>

### CODE REVIEW

**Comment by cgiradkar** [comment]
Relates to #125

**Comment by TomSweeneyRedHat** [comment]
@cgiradkar can you summarize the change(s) that you had to make?  The review page just shows completely new pages being added.

**Comment by cgiradkar** [comment]
Sure @TomSweeneyRedHat 
Here are the changes:
- All the blog posts from _posts folder (except RAs) , from old podman repo, have been moved to blog folder in this repo.
- For docusaurus to work most efficiently, all the plugins which are used should have same version, so made all plugins' version to be 2.4.1
- Change the links for blog.podman.io to podman.io/blogs

TODO: on some places, we fetch data from blog.podman.io and display, that needs to change to fetch the data locally.

**Comment by TomSweeneyRedHat** [comment]
@cgiradkar will the old links that are floating around in the internet for these pages start working again once this is merged?  Or do we need to do some DNS magic to make that happen.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM fwiw

**Comment by cgiradkar** [comment]
> @cgiradkar will the old links that are floating around in the internet for these pages start working again once this is merged? Or do we need to do some DNS magic to make that happen.

I have tested production build locally and it is working as intended. Just we needed a go ahead from @mairin for the formatting of markdown files as mentioned [here](https://github.com/containers/podman.io/issues/125#issuecomment-1686382212)

**Comment by mairin** [comment]
Hey, I left a comment on https://github.com/containers/podman.io/issues/125 - is that required to be approved before this PR can be?

**Comment by cgiradkar** [comment]
> Hey, I left a comment on #125 - is that required to be approved before this PR can be?

@mairin 125 is the issue and this is the PR

**Comment by TomSweeneyRedHat** [comment]
@cgiradkar still a WIP, or time to take the plunge with this one?

**Comment by mairin** [comment]
see comments here https://github.com/containers/podman.io/issues/125#issuecomment-1708176357

**Comment by TomSweeneyRedHat** [comment]
@cgiradkar I think this PR needs an updated yarn.lock included.

**Comment by cgiradkar** [comment]
> @cgiradkar I think this PR needs an updated yarn.lock included.

Done

**Comment by TomSweeneyRedHat** [comment]
LGTM
@mairin are you happy with #125 state?  Can this be merged?

**Comment by mairin** [comment]
yep merging!

**Comment by cgiradkar** [comment]
@TomSweeneyRedHat We still need to redirect users from podman.io/blog to podman.io/blog`s` .
